src/class-list.ts

```diff
@@ -0,0 +1,34 @@
+function _classesOf(element): string[] {
+  return element.className.trim().split(/\s+/);
+}
+
+export function add(element, ...args): void {
+  const toAdd = Array.from(args);
+  const className = _classesOf(element)
+    .filter(function (classname) {
+      return toAdd.indexOf(classname) === -1;
+    })
```

Comment:
this should be an arrow function—beyond the `this` distinction, functions that only return are like the ideal use case. also array methods.

```suggestion
    .filter((classname) => toAdd.indexOf(classname) === -1)
```